LICENSE

```diff
@@ -351,3 +351,32 @@ Files: android_jni/gradlew*
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+------------------------------------------------------------------------------
+
+Files: src/scale/*
```

Comment:
I wonder whether the new folder should contain "libyuv" in its name. It would make it clear that it exclusively contains libyuv-imported code.